pkg/collector/corechecks/cluster/ksm/kubernetes_state.md

```diff
@@ -53,6 +53,8 @@
 | `kubernetes_state.pod.volumes.persistentvolumeclaims_readonly` | Describes whether a persistentvolumeclaim is mounted read only.   | `kube_namespace` `pod_name` `volume` `persistentvolumeclaim` (`env` `service` `version` from standard labels)   |
 | `kubernetes_state.pod.unschedulable` | Describes the unschedulable status for the pod.   | `kube_namespace` `pod_name` (`env` `service` `version` from standard labels)   |
 | `kubernetes_state.pod.status_phase` | The pods current phase.   | `kube_namespace` `pod_name` `phase` (`env` `service` `version` from standard labels)    |
+| `kubernetes_state.pod.age` | The time in seconds since the creation of the pod.   | `kube_namespace` `pod_name` `phase` (`env` `service` `version` from standard labels)    |
+| `kubernetes_state.pod.uptime` | The time in seconds since the pod started.   | `kube_namespace` `pod_name` `phase` (`env` `service` `version` from standard labels)    |
```

Comment:
`uptime` might be misleading compare to the description. I think it's set when the POD is scheduled and the first container (`pause`) starts, which is not exactly similar to `uptime` for containers.

Reply:
It's the time since the pod is scheduled and acknowledged by the Kubelet, I can make the description more explicit in this direction. Let me know if you think of an alternative to `uptime`

Reply:
@vboulineau any thoughts on the change?

Reply:
The description is indeed better. Let's go with this as I don't think `uptime` has a well-defined meaning for PODs.